main.tf

```diff
@@ -5,7 +5,7 @@ terraform {
       version = ">= 3.24.1"
     }
   }
-  required_version = "~> 0.14"
+  required_version = "> 0.14"
```

Comment:
`=> 0.14` feels more correct to me, unless the tutorial explicitly doesn't work with 0.14.